src/components/Dashboard/index.js

```diff
@@ -39,6 +51,38 @@ const Dashboard = () => {
   const dispatch = useDispatch();
   const errorProp = useSelector(({ auth }) => auth.profile.error);
   const loadingProp = useSelector(({ auth }) => auth.profile.loading);
+
+  const [name, setName] = useState("");
+  const [nameValidateError, setNameValidateError] = useState(false);
+  const [nameValidateErrorMessage, setNameValidateErrorMessage] = useState("");
+  const [orgName, setOrgName] = useState("");
+  const [orgNameValidateError, setOrgNameValidateError] = useState(false);
+  const [
+    orgNameValidateErrorMessage,
+    setOrgNameValidateErrorMessage,
+  ] = useState("");
+  const [handle, setHandle] = useState("");
+  const [handleValidateError, setHandleValidateError] = useState(false);
+  const [handleValidateErrorMessage, setHandleValidateErrorMessage] = useState(
+    ""
+  );
+  const [orgHandle, setOrgHandle] = useState("");
+  const [orgHandleValidateError, setOrgHandleValidateError] = useState(false);
+  const [
+    orgHandleValidateErrorMessage,
+    setOrgHandleValidateErrorMessage,
+  ] = useState("");
```

Comment:
Move to a single line

Reply:
This is from Prettier format on save. I will disable that and get them to a single line. Thank you for mentioning that.